unstructured/partition/email.py

```diff
@@ -219,12 +240,15 @@ def partition_email(
     exactly_one(filename=filename, file=file, text=text)
 
     if filename is not None:
-        encoding, file_text = read_txt_file(filename=filename, encoding=encoding)
-        msg = email.message_from_string(file_text)
-
+        encoding, msg = parse_email(filename=filename)
```

Comment:
this is odd in that it is overwriting the `encoding` kwarg from line 227

Reply:
oh, nvm, i see `encoding` is only applicable in the `text` case. maybe renaming the variable here would make that a bit more obvious?

Reply:
Yeah I agree it would be good to rename there variable here to make that clear.

Reply:
@cragwolfe That sounds like a good point! I'll be working on renaming the variable.